src/redux/configureStore.js

```diff
@@ -0,0 +1,24 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+
+const initialState = {
+  greeting: [],
+};
+
+function rootReducer(state, action) {
+  switch (action.type) {
+    case 'GET_GREETING_SUCCESS':
```

Comment:
According to the project requirement, please ensure that you have  a store, an action , and a reducer that uses the url to the endpoint from the Rails API.